pulsar-functions/instance/src/main/java/org/apache/pulsar/functions/instance/JavaInstanceRunnable.java

```diff
@@ -205,23 +205,31 @@ public void run() {
                     }
                 }
 
-                // process the message
-                long processAt = System.currentTimeMillis();
-                stats.incrementProcessed(processAt);
                 addLogTopicHandler();
                 JavaExecutionResult result;
 
+                // set last invocation time
+                stats.setLastInvocationTime(System.currentTimeMillis());
+
+                // start time for process latency stat
+                Summary.Timer requestTimer = FunctionStats.statProcessLatency.labels(metricsLabels).startTimer();
```

Comment:
I think we should be able to set the label once in the constructor instead of each time

Reply:
I am not sure API allows for that